hex_to_binary.py

```diff
@@ -0,0 +1,14 @@
+file_path = "contracts/system-contracts/contracts-preprocessed/artifacts/EvmInterpreterPreprocessed.yul.zbin"
```

Comment:
I don't know if the PR is ready for review, but will mention just in case that we don't use python or bash for anything meaningful in the monorepo (and try to avoid them in general). In the monorepo we stick to either Rust or TypeScript.